mock-core-configs/mock-core-configs.spec

```diff
@@ -1,6 +1,10 @@
+%if 0%{?epel} == 8
+%global python3 /usr/libexec/platform-python
+%endif
+
 Name:       mock-core-configs
 Version:    38.5
-Release:    1%{?dist}
+Release:    2%{?dist}
```

Comment:
Please don't update `Release` and `%changelog` upstream, it is going to be automatically updated during the release (via `tito tag`).

Reply:
Sounds good, will fix (wasn't aware of the process for this).